test/functional/feature_dbcrash.py

```diff
@@ -75,7 +75,8 @@ def skip_test_if_missing_module(self):
     def setup_network(self):
         self.add_nodes(self.num_nodes, extra_args=self.extra_args)
         self.start_nodes()
-        self.import_deterministic_coinbase_privkeys()
+        for n in self.nodes:
+            self.import_deterministic_coinbase_privkeys()
```

Comment:
In commit "test: create default wallet in extended tests" (165b8fd5617e4bd4d5f99586306a66f084d95d13)

Could you add a comment here? It's unclear why you'd want to call the same function n times.

Reply:
That's a leftover bug from my refactor.